helmfile/values/networkpolicy-common.yaml.gotmpl

```diff
@@ -53,3 +53,27 @@ certManager:
     ips: {{- toYaml .Values.networkPolicies.certManager.letsencrypt.ips | nindent 6 }}
 falco:
   enabled: {{ and .Values.falco.enabled .Values.networkPolicies.falco.enabled }}
+
+kubeSystem:
+  enabled: {{ .Values.networkPolicies.kubeSystem.enabled }}
+
+  {{- if hasKey .Values.networkPolicies.kubeSystem "openstack" }}
```

Comment:
This is needed so that you don't need to have all these configs in the default config right?

Reply:
Yes, more or less like we do for other things that might be cloud dependant.